src/platforms/generic_unix/lib/sys.c

```diff
@@ -32,7 +32,18 @@
 #include "utils.h"
 
 #if ATOMVM_HAS_MBEDTLS
+#include <mbedtls/ctr_drbg.h>
+#include <mbedtls/entropy.h>
+
+#ifndef AVM_NO_SMP
+#include <mbedtls/config.h>
```

Comment:
mbedtls/config.h is not necessarily installed with mbedtls 3.x

Reply:
It is needed for checking `MBEDTLS_THREADING_C`.
Do you know any other alternative.

one might be GCC `#if __has_include` but maybe it is not the right choice.

Reply:
Please see mbedtls 3.0 migration guide section related to config.h.

You can check mbedtls version using C macro.

https://github.com/Mbed-TLS/mbedtls/blob/development/docs/3.0-migration-guide.md#introduce-a-level-of-indirection-and-versioning-in-the-config-files

Reply:
Fixed.